routing/bicycle_model.hpp

```diff
@@ -14,10 +14,15 @@ class BicycleModel : public VehicleModel
   BicycleModel();
   BicycleModel(VehicleModel::InitListT const & speedLimits);
 
-  /// @name Overrides from VehicleModel.
-  //@{
+  /// VehicleModel overrides:
   double GetSpeed(FeatureType const & f) const override;
-  //@}
+  bool IsOneWay(FeatureType const & f) const override;
+  /// @returns true if |f| could be considered as a road.
+  /// @note If BicycleModel::IsRoad(f) returns false for a feature f and for an instance
```

Comment:
Yes. This comment did not reflect the idea. It removed. Please have a look at new comment beside BicycleModel::IsRoad() and PedestrianModel::IsRoad() methods.